wagtail/contrib/typed_table_block/static_src/typed_table_block/scss/typed_table_block.scss

```
@@ -1,9 +1,15 @@
 @import 'wagtailadmin/scss/helpers';
 
 .typed-table-block {
+    &__wrapper {
+        overflow-x: auto;
+        // https://stackoverflow.com/questions/6421966/css-overflow-x-visible-and-overflow-y-hidden-causing-scrollbar-issue#answer-39554003
+        padding-bottom: 600px;
+        margin-bottom: -600px;
+    }
+
     table {
-        margin-top: 1.5em;
-        margin-left: 1.5em;
+        margin: 2.5em 4.5em 1em 2.5em;
```

This right margin seems to have no effect, and the bottom margin adds a little too much space before the Add Row button, in my opinion.

```suggestion
        margin: 2.5em 0 0 2.5em;
```